ci/Jenkinsfile.tests-ui

```diff
@@ -70,8 +70,10 @@ pipeline {
 
     stage('Check Translations') {
       steps {
-        script {
-          checkTranslations()
+        catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
```

Comment:
I made the TS file check optional; there's a Qt bug which doesn't catch `qsTr()` calls inside JS backticks

CC @siddarthkay @markoburcul

Reply:
Like here: https://github.com/status-im/status-desktop/blob/59bb23eef4ffec42601a439cdc6ddfd0433d1a82/ui/app/AppLayouts/Onboarding/pages/KeycardEnterPinPage.qml#L126-L129